Purchasing.Jobs.AggieEnterprise/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Purchasing.Core;
+using Purchasing.Core.Helpers;
+using Purchasing.Core.Models.Configuration;
+using Purchasing.Core.Services;
+using Purchasing.Jobs.Common;
+using Serilog;
+
+class Program : WebJobBase
+{
+    static async Task Main(string[] args)
+    {
+        var kernel = ConfigureServices();
+
+        var provider = ConfigureServicesLocal();
+
+        try
+        {
+            Log.Information("Updating Aggie Enterprise Purchasing Categories (Commodities)");
+            var aeCommodityCodeService = provider.GetService<IAePurchasingCategoryService>();
+            if(aeCommodityCodeService == null)
+            {
+                throw new Exception("Could not get the aeCommodityCodeService service");
```

Comment:
how could this ever be null?

Reply:
It should be. but I was getting nagged by the compiler